CHANGELOG.rst

```diff
@@ -4,6 +4,7 @@ Changelog
 master
 ------
 
+- (`#78 <https://github.com/znicholls/silicone/pull/78>`_) Changed how quantile rolling windows works by adding an extra interpolate step for smoothness
```

Comment:
```suggestion
- (`#78 <https://github.com/znicholls/silicone/pull/78>`_) Updated ``DatabaseCruncherQuantileRollingWindows `` to use an extra interpolate step for smoothness
```

Reply:
The maths is super-fiddly, it should be ok though.